modules/fedimint-ln-common/src/api.rs

```diff
@@ -51,6 +52,14 @@ where
     }
 
     async fn fetch_contract(&self, contract: ContractId) -> FederationResult<ContractAccount> {
```

Comment:
are we keeping the old api for back compat? should we somehow mark as deprecated?

Reply:
Old API changed to `wait_contract`. I added `fetch_contract`, but it currently isn't used anywhere. But it will be in https://github.com/fedimint/fedimint/pull/3322

Reply:
ack. looks good